.github/workflows/fly-deploy.yml

```
@@ -0,0 +1,23 @@
+name: Deploy to Fly.io
+
+on:
+  push:
+    branches:
+      - main # This means the workflow runs only when code is pushed to the main branch
```

```suggestion
      - main
```

I know we have a lot of commands in the code, but YAML is meant to be succinct. 
For new joiners this might be helpful, but I guess if you're not familiar with GitHub CI, you will need to comment every line in this case.

---

agree, no need for comments here